src/org/labkey/snd/SNDManager.java

```diff
@@ -297,28 +297,31 @@ public void updatePackage(User u, Container c, @NotNull Package pkg, @Nullable S
             errors.addRowError(new ValidationException(e.getMessage()));
         }
 
-        // If package is in use (either assigned to an event or project) then do not update the domain
-        if (!errors.hasErrors() && !((PackagesTable) pkgsTable).isPackageInUse(pkg.getPkgId()))
+        if (!errors.hasErrors())
         {
-            String domainURI = PackageDomainKind.getDomainURI(PackageDomainKind.getPackageSchemaName(), getPackageName(pkg.getPkgId()), c, u);
+            // If package is in use (either assigned to an event or project) then do not update the domain
+            if (!((PackagesTable) pkgsTable).isPackageInUse(pkg.getPkgId()))
```

Comment:
This doesn't seem like it would result in a functional difference. Is this just to make the code easier to follow and debug?

Reply:
The issue was really that lines 317-324 were not checking for errors before trying to continue on.  Instead of adding another error check down there, I just factored out the error check up here and put the rest of the function in that.  I think that is more clear for the next person who comes along.